QuizApp/QuizApp/UI/Quiz/QuizCollectionViewHeader.swift

```diff
@@ -0,0 +1,44 @@
+import UIKit
+
+class QuizCollectionViewHeader: UICollectionReusableView {
+
+    static let reuseIdentifier = String(describing: QuizCollectionViewCell.self)
+
+    private var label: UILabel!
+
+    override init(frame: CGRect) {
+        super.init(frame: .zero)
+
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func set(category: QuizCategory) {
+        label.text = category.rawValue.capitalized
```

Comment:
I have probably missed this before but do not use `rawValue` in deciding what should be in the UI. Instead, add a computed property to this enum, something like:
```
var name: String {
    switch self {
        case .movies:
            return "Movies"
        ....
    }
}
```
I know it will be the same as `rawValue`, but `rawValue` represents an identifier (ID), not something that should be on UI.